charts/neon-storage-controller/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: neon-storage-controller
+description: Neon storage controller
+type: application
+version: 1.2.1
```

Comment:
I suggest start with chart version 1.0.0

Reply:
A very reasonable place to start :-D

Fixed.